en/PlanView/PlanView.md

```diff
@@ -1,47 +1,48 @@
+The *Plan View* is used to plan autonomous missions for your Vehicle. Once the mission is planned and sent to the vehicle, you switch to the [*Fly View*](../FlyView/FlyView.md) to fly the mission.
 
-If your Vehicle supports a [GeoFence](PlanGeoFence.md) or [Rally Points](PlanRallyPoints.md) you can also set those up from the Plan View.
+If your Vehicle supports a [GeoFence](PlanGeoFence.md) or [Rally Points](PlanRallyPoints.md) you can also set those up from the *Plan View*.
 
-The image above shows a simple mission which starts with a takeoff, flies through two waypoints and then lands.
+The image above shows a simple mission which starts with a takeoff at the home position (H), flies through three waypoints and then lands on the last waypoint (i.e. waypoint 3).
 
 The steps to creating a missions are:
 
-1. Change to Plan View
-2. Add commands to the mission and edit as needed
-3. Send the mission to the vehicle
-4. Change to Fly View and fly your mission
+1. Change to *Plan View*.
+2. Add waypoints or commands to the mission and edit as needed.
+3. Upload the mission to the vehicle.
+4. Change to *Fly View* and fly your mission.
 
 ## Plan Tools
-On the left edge of the screen you will see the Plan Tools. The order of tools from top to bottom is:
+On the left edge of the screen you will see the Plan Tools as marked in red in the image above. 
 
-* Add Commands
-* [Survey](Survey.md)
-* Sync
-* Center map
-* Map Type
-* Zoom In/Out
+ > **Note** **Center map**, **Zoom In**, **Zoom Out** are just map visualisation tools that help user view the *Plan view* map clearly while planning missions (they don't affect the mission commands sent to the vehicle in any way).
 
-### Add Commands
-Click to activate the Add Commands tool. While active, clicking in the map will add new mission commands at the clicked location. The tool will stay active until you click it again.
+
+### Add Waypoints
+Click to activate the **Add Waypoint** tool. While active, clicking in the map will add new mission waypoint at the clicked location. The tool will stay active until you click it again. Once you have added a waypoint, you can select it by clicking on it and then drag it around to change its position.
+
+### Actual vs Planned Home Position
+Home position is the position the vehicle will return to and land on when in Return (to Launch/Home) mode. The home position shown in *Plan View* is merely a virtual home position a.k.a "Planned Home Position" which helps you simulate the vehicle's home position while planning a mission. Hence, you should place it where you roughly plan to start (takeoff) the vehicle from. This allows QGC to estimate mission times and to draw waypoint lines correctly to the home position. When you go out on the field and start the mission, the autopilot then replaces the mission's planned home position with the actual takeoff position as its home position. This actual home position can then be seen in the *Fly View*.
```

Comment:
1. I don't always do this myself, but often good to put sentences on separate lines. That makes it easier to see changes later, and is also a good split of docs for translators.
2. Suggest replace "virtual home position" with "placeholder home position". It is more clear that this is something that will be updated/replaced at some later point.

Reply:
You are going to be better off not using terminology which isn't used on the screen.

> The home position shown in *Plan View* is merely a virtual home position a.k.a "Planned Home Position" which helps you simulate the vehicle's home position while planning a mission.

"The home position shown in *Plan View* is the "Planned Home Position". It is not the actual home position which will be used when flying the mission." ...